install_files/ansible-base/roles/install_fpf_repo/tasks/main.yml

```diff
@@ -3,9 +3,15 @@
   apt_key: >
     state=present
     data="{{ lookup('file', 'fpf-signing-key.pub') }}"
+    keyring=/etc/apt/trusted.gpg.d/securedrop-keyring.gpg
```

Comment:
Just came back to double check that this file is raw instead of ASCII-armored. That results in issues. Looks like you already figured that out. 👍